### PR TITLE
Add both data directories to docker run commands

### PR DESCRIPTION
**Pull request by kmoscoe**

### CODE REVIEW

File: custom_dc/custom_data.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ In one terminal window, from the root directory, run the following command to st
 docker run \
 --env-file $PWD/custom_dc/env.list \ \
```
Comment (keyurva) [inline custom_dc/custom_data.md:264]: Let's remove the extraneous backslash while you're at it.

**Review by keyurva — APPROVED** [review]
Thanks for the quick turnaround!

File: custom_dc/custom_data.md (hunk 1 above)
Comment (kmoscoe) [reply custom_dc/custom_data.md:264]: Done.

Comment (keyurva) [reply custom_dc/custom_data.md:264]: Doesn't look like it got updated. Did you forget to push?

**Comment by kmoscoe** [comment]
Not sure what happened here. I did push. Anyway, created another PR. Please
approve.

**Pull request merged**